src/sys.zig

```diff
@@ -345,18 +345,23 @@ pub fn Maybe(comptime ReturnTypeT: type) type {
 
 pub fn getcwd(buf: *bun.PathBuffer) Maybe([]const u8) {
     const Result = Maybe([]const u8);
-    buf[0] = 0;
-    const rc = std.c.getcwd(buf, bun.MAX_PATH_BYTES);
-    return if (rc != null)
-        Result{ .result = std.mem.sliceTo(rc.?[0..bun.MAX_PATH_BYTES], 0) }
-    else
-        Result.errnoSys(0, .getcwd).?;
+    return switch (getcwdZ(buf)) {
+        .err => |err| Result{ .err = err },
+        .result => |cwd| Result{ .result = cwd },
+    };
 }
 
 pub fn getcwdZ(buf: *bun.PathBuffer) Maybe([:0]const u8) {
     const Result = Maybe([:0]const u8);
     buf[0] = 0;
```

Comment:
is this necessary?

Reply:
I don't think so, but it does ensure that even if it returns an empty string, the buffer is zero sentineled